pippo-session-ehcache2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
You are right, my mistake, originally I do individual projects in Github and for each one create a LICENSE file, I forgot delete it when I copied the projects in the main project